src/test/kotlin/uk/gov/dluhc/registercheckerapi/config/LocalStackContainerConfiguration.kt

```diff
@@ -32,10 +33,15 @@ class LocalStackContainerConfiguration {
         const val DEFAULT_PORT = 4566
     }
 
+    @Bean
+    fun awsRegionProvider(
+        @Value("\${spring.cloud.aws.region.static}") region: String,
+    ): AwsRegionProvider = StaticRegionProvider(region)
+
     @Bean
     fun awsBasicCredentialsProvider(
-        @Value("\${cloud.aws.credentials.access-key}") accessKey: String,
-        @Value("\${cloud.aws.credentials.secret-key}") secretKey: String,
+        @Value("\${spring.cloud.aws.credentials.access-key}") accessKey: String,
+        @Value("\${spring.cloud.aws.credentials.secret-key}") secretKey: String,
```

Comment:
most other repos have these set in the integration-test yaml config file and then also in the companion object in this file; I think this is better